cmd/gb-vendor/main.go

```diff
@@ -31,23 +31,26 @@ var commands = []*cmd.Command{
 }
 
 func main() {
-	root, err := cmd.FindProjectroot(projectroot)
-	if err != nil {
-		gb.Fatalf("could not locate project root: %v", err)
-	}
-	project := gb.NewProject(root)
-	gb.Debugf("project root %q", project.Projectdir())
-
 	args := os.Args[1:]
-	if len(args) < 1 || args[0] == "-h" {
+
+	switch {
+	case len(args) < 1, args[0] == "-h", args[0] == "-help":
 		fs.Usage()
 		os.Exit(1)
-	}
-
-	if args[0] == "help" {
+	case projectroot == "":
+		gb.Fatalf("don't run this binary directly, it is meant to be run as 'gb vendor ...'")
+	case args[0] == "help":
```

Comment:
This logic never fires,

zapf(~/src/github.com/constabulary/gb/cmd) % gb-vendor help
FATAL don't run this binary directly, it is meant to be run as 'gb vendor ...'

I think if you move it above the `case projectroot == ""` block it will work